presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -284,6 +284,9 @@ protocol::TaskStatus PrestoTask::updateStatusLocked() {
       info.taskStatus.physicalWrittenDataSizeInBytes,
       physicalWrittenBytes);
 
+  info.taskStatus.outputBufferUtilization = taskStats.outputBufferUtilization;
```

Comment:
Can you update all the places that we update buffer utilizations?
such as 
  prestoTask->info.taskStatus.outputBufferUtilization = 1;
in findOrCreateTaskLocked?